### ISSUE TITLE
bioSurvey flattening updates: non-existing variables

### ISSUE THREAD

**Issue by KELSEYDOWLING7** [Analyticsphere/flatteningRequests#50]
-It looks like some variables are in the flattened table that don't appear in the DD. They look like slight variations of existing concept IDs so I'm thinking they're just a typo somehow. Can these be deleted? I confirmed that for these variables there's no participant that answered yes, it's a column of all 0's.

1. D_542661394_D_104430600
2. D_542661394_D_167336300
3. D_542661394_D_178420300
4. D_542661394_D_181769800
5. D_542661394_D_215662700
6. D_542661394_D_329536000
7. D_542661394_D_371726200
8. D_542661394_D_656498900

-D_542661394_D_16733625 is a new variable that has now been answered. In the raw table but not yet the flattened table
-D_169509213_1_1, D_169509213_2_2 are in the flattened table but not the raw data. These variables have been deprecated and have no data. Can they be deleted?

**Comment by jacobmpeters** [comment]
These variables ending in `00` are due to a tricky bug in my flattening script. I'm looking into this one now..

**Comment by jacobmpeters** [comment]
I fixed a bug in the `get_unique_values` func that was converting those responses from `numeric` to `character`. Essentially the values were being represented in scientific notation and converting them was dropping the last digits. I fixed this issue and reran the script, dropping all of the extra variable names ending in `000`.

<img width="245" alt="Screenshot 2024-04-01 at 3 38 12 PM" src="https://github.com/Analyticsphere/flatteningRequests/assets/7967451/f10529f3-9e06-42de-a677-bc932c92cd94">
